lib/coverband/adapters/hash_redis_store.rb

```diff
@@ -119,7 +119,7 @@ def add_coverage_for_file(data_from_redis, hash)
 
         data = coverage_data_from_redis(data_from_redis)
         hash[file] = data_from_redis.select { |meta_data_key, _value| META_DATA_KEYS.include?(meta_data_key) }.merge!("data" => data)
-        hash[file][LAST_UPDATED_KEY] = hash[file][LAST_UPDATED_KEY].blank? ? nil : hash[file][LAST_UPDATED_KEY].to_i
+        hash[file][LAST_UPDATED_KEY] = (hash[file][LAST_UPDATED_KEY].nil? || hash[file][LAST_UPDATED_KEY] == '') ? nil : hash[file][LAST_UPDATED_KEY].to_i
```

Comment:
Yeah, a good call we do try to avoid Rails methods, this slipped in at some point... I guess most of our users are on Rails so we missed it.